notebooks/problem-sets/multi-qubit_states.ipynb

```diff
@@ -0,0 +1,380 @@
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: .option -->\n",
+    "\n",
+    "1. $\\sigma^x$\n",
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: .option -->\n",
+    "\n",
+    "2. $\\sigma^z$\n",
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: .option(x) -->\n",
+    "\n",
+    "3. $H$\n",
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: -->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Problem 5\n",
+    "\n",
+    "<!-- ::: q-quiz(goal=\"multi-qubit-states-5\") -->\n",
+    "<!-- ::: .question -->\n",
+    "\n",
+    "Let's consider $|\\beta_{00}\\rangle = \\frac{1}{\\sqrt{2}} (|00\\rangle +|11\\rangle)$\n",
+    "\n",
+    "$\\sigma^x \\otimes I |\\beta_{00}\\rangle = $\n",
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: .option(x) -->\n",
+    "\n",
+    "1. $\\frac{1}{\\sqrt{2}} (|10\\rangle +|01\\rangle)$\n",
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: .option -->\n",
+    "\n",
+    "2. $\\frac{1}{\\sqrt{2}} (|00\\rangle -|11\\rangle)$\n",
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: .option -->\n",
+    "\n",
+    "3. $\\frac{1}{\\sqrt{2}} (|00\\rangle +|11\\rangle)$\n",
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: -->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Problem 6\n",
+    "\n",
+    "<!-- ::: q-quiz(goal=\"multi-qubit-states-6\") -->\n",
+    "<!-- ::: .question -->\n",
+    "\n",
+    "Let's consider $|\\beta_{00}\\rangle = \\frac{1}{\\sqrt{2}} (|00\\rangle +|11\\rangle)$\n",
+    "\n",
+    "$\\sigma^x \\otimes \\sigma^x |\\beta_{00}\\rangle = $\n",
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: .option -->\n",
+    "\n",
+    "1. $\\frac{1}{\\sqrt{2}} (|10\\rangle +|01\\rangle)$\n",
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: .option -->\n",
+    "\n",
+    "2. $\\frac{1}{\\sqrt{2}} (|00\\rangle -|11\\rangle)$\n",
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: .option(x) -->\n",
+    "\n",
+    "3. $\\frac{1}{\\sqrt{2}} (|00\\rangle +|11\\rangle)$\n",
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: -->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Problem 7\n",
+    "\n",
+    "Create a `QuantumCircuit` that creates the $|\\beta_{00}\\rangle = \\frac{1}{\\sqrt{2}} (|00\\rangle +|11\\rangle)$"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "goals": [
+     {
+      "id": "multi-qubit-states-7"
+     }
+    ],
+    "grader_answer": "qc",
+    "grader_id": "textbook_practice_exam/multi-qubit_states/problem7"
```

Comment:
the `grader_id` should be in the form of `<event or exam>/<problem id>` so therefore this should be
`textbook_practice_exam/problem7` or better yet to avoid possible conflict with other problems in `textbook_practice_exam` maybe `textbook_practice_exam/multi-qubit-problem7` would be a better `grader_id` value